stl/src/filesys.cpp

```diff
@@ -34,7 +34,7 @@ static file_type _Map_mode(int _Mode) { // map Windows file attributes to file_s
 }
 
 _FS_DLL void __CLRCALL_PURE_OR_CDECL _Close_dir(void* _Handle) { // close a directory
-    FindClose((HANDLE) _Handle);
+    FindClose(static_cast<HANDLE>(_Handle));

```

Comment:
Per https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#HANDLE, `HANDLE` is an alias for `void*`, so this cast is extraneous. Why don't we simply remove the cast instead of transforming it into an extraneous C++-style cast? (Also on line 65.)